From 7b1586d7d7969a7fc843f411e08d32133d934ddf Mon Sep 17 00:00:00 2001
From: Chase Maupin <Chase.Maupin@ti.com>
Date: Fri, 21 Oct 2011 10:05:44 -0500
Subject: [PATCH] omap_hsmmc: make default dto value 14

* It seems that making the dto value makes all the various SD
  cards work.  If not we will sometimes see -110 errors during
  boot with certain SD cards.
* Based on input from Steve Kipisz

Upstream-Status: Pending

Signed-off-by: Chase Maupin <Chase.Maupin@ti.com>
---
 drivers/mmc/host/omap_hsmmc.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/drivers/mmc/host/omap_hsmmc.c b/drivers/mmc/host/omap_hsmmc.c
index 35505bd..643b340 100644
--- a/drivers/mmc/host/omap_hsmmc.c
+++ b/drivers/mmc/host/omap_hsmmc.c
@@ -1574,6 +1574,9 @@ static void set_data_timeout(struct omap_hsmmc_host *host,
 			dto = 14;
 	}
 
+    // Use 14 by default because this seems to work will with all cards
+    dto = 14;
+
 	reg &= ~DTO_MASK;
 	reg |= dto << DTO_SHIFT;
 	OMAP_HSMMC_WRITE(host->base, SYSCTL, reg);
-- 
1.7.0.4

